recipe/meta.yaml

```diff
@@ -55,23 +55,25 @@ requirements:
         - geos
         - proj.4
         - xerces-c
-        - libnetcdf
+        - libnetcdf 4.3*
         - krb5
-        - libtiff
         - libpng
-        - openjpeg
+        - openjpeg  # [not py35]
+        - libtiff 4.0.6  # [not py35]
```

Comment:
`libtiff 4*` ends up getting the default channel `libtiff 4.0.2` and that is not working on OS X.